content/millennium/r4/financial/general/charge-item.md

```diff
@@ -212,6 +212,61 @@ The `ETag` response header indicates the current `If-Match` version to use on su
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Operation: charge-item-create
+
+<%= beta_tag(action: true) %>
+
+Creates a charge event which can result in the creation of one or more ChargeItems.
+
+    POST /ChargeItem/:id/$create
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/fhir+json', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:charge_item_create, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.cerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/ChargeItem/$create
```

Comment:
The tenant key provided here should be from prod. Currently, it is from dev.

Reply:
I believe ec2458f2-1e24-41c8-b71b-0e701af7583d is the C1941 tenant that should be used.

Reply:
Updated all where applicable  [Fixed](https://github.com/cerner/fhir.cerner.com/pull/778/files#diff-2830ed2523973e82037a23daf274840e05a6c6e05b9f698fbd5f3e7fc24f57bbR245)